### DISCUSSION TITLE
How to display inner edges of volume cells?

### DISCUSSION THREAD

**Discussion by zhang9song** [pyvista/pyvista#2581] [Q&A] [answered]
Hi, guys

Is possible to display inner edges of volume cells under wireframe?  The opacity parameter is for mesh.

**Comment by banesullivan** [comment, marked as answer]
Best way is to pre-extract all edges and plot separately:

```py
import pyvista as pv
vol = pv.Wavelet()

p = pv.Plotter(notebook=0)
p.add_volume(vol)
p.add_mesh(vol.extract_all_edges(), color='k')
p.show()
```
![Screen Shot 2022-06-27 at 6 38 18 PM](https://user-images.githubusercontent.com/22067021/176062457-4ddee736-e29c-4675-9b7a-ab6bac0f0229.png)

**Reply by zhang9song** [reply]
Thank for your help.
